src/components/subcomponents/registrationValidation/password-validation.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Typography, Grid } from "@material-ui/core";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import CancelIcon from "@material-ui/icons/Cancel";
+import { makeStyles } from "@material-ui/core/styles";
+
+export const passwordSecureCheck = (password, confirmPassword) => {
+  return null;
+};
+
+const useStyles = makeStyles(theme => ({
+  pass: {
+    color: "#4caf50",
+    verticalAlign: "middle"
+  },
+  fail: {
+    color: "#d50000",
+    verticalAlign: "middle"
+  },
+  margin: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1)
+  },
+  icon: {
+    fontSize: "1.3rem",
+    marginRight: theme.spacing(1),
+    verticalAlign: "text-bottom"
+  }
+}));
+
+export function PasswordValidation({ password, confirmPassword }) {
+  const classes = useStyles();
+  const length = password.length >= 8;
+  const capital = /[A-Z]/.test(password);
+  const number = /[0-9]/.test(password);
+  const special = /[!@#$%&*?]/.test(password);
+  const confirm = (confirmPassword === password && length);
```

Comment:
I like this a lot, storing the test condition in variables and later use the variable to validate

Each variables make sense with the condition that they are testing.

```javascript
{conditionIcon(length)}Password must have a minimum of 8 characters
```